state-sync/data-streaming-service/src/data_stream.rs

```diff
@@ -1723,20 +1671,9 @@ async fn subscribe_to_transactions_with_proof<
         request.include_events,
         request_timeout_ms,
     );
-    client_response.await.map(|response| {
-        let (context, (transaction_list_with_proof_v2, ledger_info_with_signatures)) =
-            response.into_parts();
-        let transaction_list_with_proof =
-            transaction_list_with_proof_v2.get_transaction_list_with_proof();
-        let response_v1 = Response::new(
-            context,
-            (
-                transaction_list_with_proof.clone(),
-                ledger_info_with_signatures,
-            ),
-        );
-        response_v1.map(ResponsePayload::from)
-    })
+    client_response
```

Comment:
This is undoing the changes made in previous PR (https://github.com/aptos-labs/aptos-core/pull/17056/files). Is this intendend?